src/configs/lang.ts

```diff
@@ -602,3 +603,25 @@ export const CUSTOMIZE_COLUMNS_FILTER_MESSAGE = translate(
   'CUSTOMIZE_COLUMNS_FILTER_MESSAGE',
   'Select columns to be displayed'
 );
+
+export const DRAFT_PLANS = translate('DRAFT PLANS', 'Draft plans');
+
+export const AN_ERROR_OCURRED = translate(
+  'AN_ERROR_OCURRED',
+  'An error ocurred. Please try and refresh the page.'
+);
```

Comment:
Didn't we already merge a PR that had some of these strings?

Reply:
will remove this during the rebase